### PR TITLE
Validator Content Reorganization

### PR DESCRIPTION
**Pull request by DannyS03**
- changes how the content is presented
- reorganization of sidebar
- creates section for delegating (knowledge & further how-tos can be added as a follow up to this PR)
- creates section for governance (knowledge & further how-tos can be added as a follow up to this PR)
- will combine a few docs and remove duplications
- ~will limit the number of docs only used to reference or point directly to another doc~ will do in follow up PR once section design is determined

### CODE REVIEW

**Comment by DannyS03** [comment]
Current Iteration:

image

File: docs/maintain/validator/responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  - docs
+  - matic
+  - polygon
+  - validator
+image: https://matic.network/banners/matic-network-16x9.png 
+---
+
+## Overview
+
+For the detailed description on what is a validator, see [Validator](/docs/validate/validator/introduction).
```
Comment (cerberushades) [inline docs/maintain/validator/responsibilities.md:15]:
```suggestion
For the detailed description on what a validator is, see [Validator](/docs/validate/validator/introduction).
```

File: docs/maintain/govern/governance.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+keywords:
+  - docs
+  - matic
+  - polygon
+  - DAO
+  - governance
+image: https://matic.network/banners/matic-network-16x9.png 
+---
+
+<!-- This page is a WIP
```
[the remark this answers is not in the document]
Comment (DannyS03) [reply docs/maintain/govern/governance.md:14]: Added that the page is under construction

[the remark this answers is not in the document]
Comment (DannyS03) [reply docs/maintain/govern/governance.md:14]: As a notice, it is a good way to inform of upcoming content and also allow us to restructure

File: sidebars.js [hunk 3]
```diff
@@ -525,74 +525,101 @@ module.exports = {
...
       type: "category",
-      label: "Overview",
+      label: "Knowledge Base",
       items: [
-        "validate/validator/introduction",
-        "validate/validator/architecture",
+        "maintain/polygon-basics/what-is-polygon",
+        "maintain/polygon-basics/what-is-proof-of-stake",
+        "maintain/polygon-basics/who-is-delegator",
+        "maintain/polygon-basics/who-is-validator",
```
[the remark this answers is not in the document]
Comment (DannyS03) [reply sidebars.js:536]: right there are two pages that try to address the same thing, deleting the intro one

**Review by cerberushades — COMMENTED** [review]
Hey @DannyS03 i've review your PR but we still need to reach a consensus on the "Maintain" issue. Should we put this PR on hold then @Nazeim ?

**Comment by Nazeim** [comment]
@DannyS03 
Let's do 1 more modification and then we can merge.

There is kind of duplication in the article **/docs/maintain/polygon-basics/who-is-delegator** and **/docs/maintain/validator/core-components/delegation**

**Comment by DannyS03** [comment]
Most recent commit also fixes #871

**Comment by vcastellm** [comment]
Can you provide a test live link to explore the changes? if not, can you post the structure again in a screenshot? Thx!

File: docs/maintain/validate/install-gcp.md [hunk 4]
```diff
@@ -5,14 +5,17 @@ sidebar_label: Google Cloud simple deploy
 description: Simple deployment of your Polygon nodes in Google Cloud
```
Comment (vcastellm) [inline docs/maintain/validate/install-gcp.md:5]: This is related to Integrate -> Full node deployment, can we move there?

**Review by vcastellm — COMMENTED** [review]
Why `Maintain` instead of `Validate`?

**Comment by DannyS03** [comment]
> Why `Maintain` instead of `Validate`?

The idea is to capture the other docs that aren't specifically about validating (i.e. delegation and governance, which will include more content soon). This also extends to the validator docs themselves as they are guides about maintaining the network as a validator, and not only how-tos on how to validate

Some ways others capture this on their docs is `Node`, `Stake` or `Staking`, `Participate`, but `Maintain` is best suited.

File: docs/maintain/validate/install-gcp.md (hunk 4 above)
Comment (DannyS03) [reply docs/maintain/validate/install-gcp.md]: Updated

**Review by cerberushades — APPROVED** [review]

**Review by cerberushades — APPROVED** [review]

**Pull request merged**